docs/esa/hubble/hubble.rst

```diff
@@ -545,6 +545,54 @@ returns the corresponding HAP or HST observation
   ['jec071i9q']
 
 
+-----------------------------------------------------------
+10. Retrieve metadata and data from a program / proposal ID
+-----------------------------------------------------------
+
+It is possible to retrieve all the observations associated to a specific program ID using the following method:
+
+.. doctest-remote-data::
+
+  >>> from astroquery.esa.hubble import ESAHubble
+  >>> esahubble = ESAHubble()
+  >>> esahubble.get_observations_from_program(program=5773)
+  <Table length=68>
+  band_name calibration_level collection ...      wave_max      wave_min
+  object        int32         str64    ...      float64       float64
+  --------- ----------------- ---------- ... ------------------ --------
+  Optical                 2        HST ... 502.58000000000004   499.89
+  Optical                 2        HST ...  570.9399999999999   522.62
+  Optical                 2        HST ...             675.58   670.86
+  Optical                 2        HST ...             657.46    655.3
+  Optical                 2        HST ...  570.9399999999999   522.62
+  Optical                 1        HST ... 502.58000000000004   499.89
+  Optical                 2        HST ...             657.46    655.3
+  Optical                 1        HST ...             657.46    655.3
+  Optical                 2        HST ... 502.58000000000004   499.89
+  Optical                 1        HST ...             675.58   670.86
+      ...               ...        ... ...                ...      ...
+  Optical                 3        HLA ...             570.94   522.62
+  Optical                 3        HLA ...             570.94   522.62
+  Optical                 3        HLA ...             570.94   522.62
+  Optical                 3        HLA ...             570.94   522.62
+  Optical                 3        HLA ... 502.58000000000004   499.89
+  Optical                 3        HLA ... 502.58000000000004   499.89
+  Optical                 3        HLA ... 502.58000000000004   499.89
+  Optical                 3        HLA ... 502.58000000000004   499.89
+  Optical                 3        HLA ... 502.58000000000004   499.89
+  Optical                 3        HLA ... 502.58000000000004   499.89
```

Comment:
whitespace is off here failing this test.

Btw, the latest version of `pytest-doctestplus` has a new feature to auto-update the doctests, you may find it useful for cases like this: 
command: `pytest docs/esa/hubble/hubble.rst --doctest-plus-generate-diff=overwrite -R`
more info: https://github.com/scientific-python/pytest-doctestplus/blob/main/README.rst#fixing-existing-docstrings

```suggestion
  <Table length=68>
  band_name calibration_level collection ...      wave_max      wave_min
    object        int32         str64    ...      float64       float64
  --------- ----------------- ---------- ... ------------------ --------
    Optical                 2        HST ... 502.58000000000004   499.89
    Optical                 1        HST ...             657.46    655.3
    Optical                 2        HST ...             657.46    655.3
    Optical                 2        HST ...             675.58   670.86
    Optical                 2        HST ... 502.58000000000004   499.89
    Optical                 2        HST ... 502.58000000000004   499.89
    Optical                 2        HST ...  570.9399999999999   522.62
    Optical                 1        HST ...  570.9399999999999   522.62
    Optical                 2        HST ...             657.46    655.3
    Optical                 2        HST ...             675.58   670.86
        ...               ...        ... ...                ...      ...
    Optical                 3        HLA ...             570.94   522.62
    Optical                 3        HLA ...             570.94   522.62
    Optical                 3        HLA ...             570.94   522.62
    Optical                 3        HLA ...             570.94   522.62
    Optical                 3        HLA ... 502.58000000000004   499.89
    Optical                 3        HLA ... 502.58000000000004   499.89
    Optical                 3        HLA ... 502.58000000000004   499.89
    Optical                 3        HLA ... 502.58000000000004   499.89
    Optical                 3        HLA ... 502.58000000000004   499.89
    Optical                 3        HLA ... 502.58000000000004   499.89
```

Reply:
I finally removed it, as the result may change.

Reply:
And many thanks for the command, for sure we will use it.